controller/src/main/java/io/pravega/controller/server/rpc/grpc/GRPCServerConfig.java

```diff
@@ -35,4 +35,39 @@
      * @return The RPC port which has to be registered with the cluster used for external access.
      */
     Optional<Integer> getPublishedRPCPort();
+
+    /**
+     * Fetches the settings which indicates whether authorization is enabled.
+     *
+     * @return Whether this deployment has auth enabled.
+     */
+    boolean isAuthorizationEnabled();
```

Comment:
This seems like a dangerous setting. Particularly if it has a default.
Perhaps we could have a setting like authenticationMethod which is an enum and and something like "OpenAccess" is one of the options.